### ISSUE TITLE
Go Live Support

### ISSUE THREAD

**Issue by jomentallyfriendly** [MentallyFriendly/ensi#184]
- [ ] What is the recommended stand by time for launch if we **do not** handle the domains?
- [ ] What is the recommended stand by time for launch if we **do** handle the domains?

**Comment by smypf** [comment]
Standby as in the time that will be needed to do the work?

Moving the domains will be a long process. I'd estimate at least 30 mins per domain to move these. Ideally we wouldn't be the ones that do this.

Otherwise standby will be minimal if we're not working on the domains, as we'd only need to set up the domains for the site and not the actual settings. 2 hours max.

**Comment by jomentallyfriendly** [comment]
Nope. I know about the domain effort. If we don't do the domains and are just there if anything goes wrong on launch (launch = their IT configures the domains) would you think we need to be waiting for an issue all day or just for half a day? I just try to gauge how much can go wrong. I personally think we should be fine once everything works on Webflow so tend to offer them 4hrs stand by time.

**Comment by smypf** [comment]
Not much can go wrong as far as I can see. It's not like when we have to
deploy a site and make sure that all the services work as would be
expected.​ Your estimate seems appropriate.
